LawnCare/frontend/app/src/main/java/com/redflag/lawncare/MainActivity.java

```diff
@@ -12,8 +12,11 @@
 import com.redflag.lawncare.common.TabsAdapter;
 import com.redflag.lawncare.settings.SettingsActivity;
 
+import java.util.Locale;
+
 
 public class MainActivity extends AppCompatActivity {
+    protected Locale french = new Locale("fr");
```

Comment:
why?

Reply:
Oh whoops, I was playing around to see how I could change the text of tabs, I didn't catch it when I was deleting everything